conf/supervisord.conf

```diff
@@ -0,0 +1,19 @@
+[supervisord]
+nodaemon=true
+environment=PYTHONUNBUFFERED="1"
+logfile=/dev/null
+logfile_maxbytes=0
+
+[program:nginx]
+command=nginx -g 'daemon off;' 
+stdout_logfile=NONE
+stderr_logfile=NONE
+stdout_events_enabled=true
+stderr_events_enabled=true
+
+[program:web]
+command=gunicorn -b 0.0.0.0:9082 py_proxy.app:app() -b unix:/tmp/gunicorn-web.sock
```

Comment:
We've just recently had to change `supervisord.conf` in h and lms to run pserve instead of gunicorn, due to a gunicorn upgrade: https://github.com/hypothesis/lms/blob/e7a72fdf884d924bbc7f7f783e3bea470376a1eb/conf/supervisord.conf#L8

Reply:
Do I have to add a prod.ini file for this? It kinda looks that way but I wasn't sure. Anyways-didn't get to this today. I'll see what I can do about this tomorrow.

Reply:
Not sure. Depends whether `pserve` requires one, I guess. Actually, now that I think about it, bouncer doesn't have any `.ini` files and as a result it [gets away with calling `gunicorn` directly](https://github.com/hypothesis/bouncer/blob/966117c5fb8e2922b6b14e8094d2a1afc2d6ed6c/conf/supervisord.conf#L8) (since ini file parsing is the reason why `pserve` is needed). So if the new app doesn't need an ini file this should be fine.